### ISSUE TITLE
Screen readers (NVDA/JAWS) are not announcing required information for the required 'Title' edit field.

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#573]
Note: Same issue reproes throughout the application wherever the similar edit fields are present.

Other Instances:
Other instance 1: Votes tab
Path: Retrospective page-->Votes tab
Issue: NVDA/JAWS are not narrating required information for the 'Feedback title' edit field, while navigating using browse/cursor mode. Ex: it is narrating as "test", it should narrate as "Feedback title is test. Click to edit. paragraph  required"
PFA (Other Instance_1_NVDAJAWS are not narrating required information for the 'Feedback title' edit field, while navigating using browse/cursor mode.)
Note: Same issue is observed wherever the similar edit fields are present throughout the application.

Impact on a Person with Disability (PWD):
Users who rely on screen readers are facing difficulty and unaware of the required fields as Screen readers (NVDA/JAWS) are not announcing required information for the required 'Title' edit field.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Screen Readers: NVDA 2023.1, JAWS (2023.2303.144)

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to '...' ellipsis control and invoke it.
Step 4: Navigate to Create new retrospective button and invoke it.
Step 5: Create new retrospective Dialog gets appear.
Step 6: Navigate to 'Title' edit field.
Step 7: Verify whether Screen readers (NVDA/JAWS) are announcing required information for the required 'Title' edit field or not.

Actual Result: 
Screen readers (NVDA/JAWS) are not announcing required information for the required 'Title' edit field.
Current Observation: NVDA is announcing as 'Please enter new retrospective name edit selected Testing 4'.

Expected Result: 
Screen readers (NVDA/JAWS) should announce required information for the required 'Title' edit field.
Ex: NVDA should announcing as 'Title edit required Please enter new retrospective name edit selected Testing 4'.

Suggestive Fixes:
1.Provide 'aria-required="true"' attribute for the 'Title'
edit field.
2.Repeat same procedure for remaining instances.

MAS Reference: 
[MAS 3.3.2 – Labels or Instructions](https://nam06.safelinks.protection.outlook.com/ap/w-59584e83/?url=https:%2f%2fmicrosoft.sharepoint.com%2f:w:%2fr%2fsites%2faccessibility%2f_layouts%2f15%2fDoc.aspx?sourcedoc%3D%257B3752B546-30D8-44D7-8221-6FC32EEB3AC8%257D%26file%3DMAS%25203.3.2%2520%25E2%2580%2593%2520Labels%2520or%2520Instructions.docx%26wdLOR%3Dc36957864-4B45-43F1-AB1E-E2E3A9834EFC%26action%3Ddefault%26mobileredirect%3Dtrue&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671699303779%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=FnZ8bW%2fmU4IGRomcx2dWTitGBuK25fKrHbuVdmZIAUw%3D&reserved=0)

WCAG Reference:
[https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.w3.org%2fWAI%2fWCAG21%2fUnderstanding%2flabels-or-instructions.html&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671699303779%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=QToQIVVSI7%2f3DxIHjkoY4X%2fOygw8Wwjj7gkVLqq3UMo%3D&reserved=0)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713036
